### PR TITLE
fix: Profile dropdown tippy

### LINKED ISSUE #590: Can have better approach?
image

i personally dont like this scroll to reveal logic. Any better ideas?

### PR DESCRIPTION
**Pull request by tokitou-san**
**Note**: Dont merge this now
Closes #590

### CODE REVIEW

**Comment by tokitou-san** [comment]
@baseplate-admin  Do you like this gradient background or check preview

Screenshot from 2023-09-08 10-10-59

**Comment by tokitou-san** [comment]
This half thing is an UX technique, so hovering probability is higher :)

Screenshot from 2023-09-08 11-14-20

**Comment by baseplate-admin** [comment]
image

File: src/lib/components/tippies/profile_dropdown.svelte [hunk 1]
```diff
@@ -58,15 +58,15 @@
...
             Tokito_Muichiro_is_best#0001
         </HoverExpand>
         <HoverExpand
-            class="text-xs text-surface-100 md:text-[0.8vw] md:leading-none"
-            height="md:max-h-[1.35vw] md:hover:max-h-[10vw]"
+            class="text-xs text-surface-100 break-words md:text-[0.8vw] md:leading-none"
+            height="md:max-h-[0.85vw] md:hover:max-h-[10vw]"
         >
-            sora_amamiya@coreproject.moe
+            tokito_muichiro_is_best@coreproject.moe
```
Comment (baseplate-admin) [inline src/lib/components/tippies/profile_dropdown.svelte:69]: I dont see the name

Comment (tokitou-san) [reply src/lib/components/tippies/profile_dropdown.svelte:69]: Sheldon Look at me

Comment (baseplate-admin) [reply src/lib/components/tippies/profile_dropdown.svelte:69]: > Sheldon Look at me
 
image

File: src/routes/anime/+layout.svelte [hunk 2]
```diff
@@ -185,12 +185,13 @@
...
                                 allowHTML: true,
                                 placement: "bottom-end",
-                                offset: [0, 10],
                                 animation: "shift-away",
                                 appendTo: document.body,
                                 interactive: true,
                                 theme: "elaine",
                                 onTrigger: async (instance) => {
+                                    // set "1 vw"
+                                    instance.props.offset = [0, (1 * window.innerWidth) / 100]
```
Comment (baseplate-admin) [inline src/routes/anime/+layout.svelte:194]: This should be calculated better. Add a `vw` fucntion

Comment (tokitou-san) [reply src/routes/anime/+layout.svelte:194]: Yea, are you sure? this is basic and maybe we dont use this more often

Comment (baseplate-admin) [reply src/routes/anime/+layout.svelte:194]: Yea cause we might need this later

**Pull request merged**